src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -2,18 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Buffers;
-using System.Buffers.Text;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Runtime.InteropServices;
 
 namespace System.Text.Json
 {
     /// <summary>
     ///   Represents a specific JSON value within a <see cref="JsonDocument"/>.
     /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public readonly partial struct JsonElement
```

Comment:
Should Document have something? Maybe the `RootElement`?

Reply:
I didn't see how I could do that in a useful way. @bartonjs?

Reply:
I, personally, don't see much value in putting a debugger display on JsonDocument. (Though, I'd also be fine with leaving JsonElement and JsonDocument as not having a debugger display, and just using the debugger's default of ToString())